.travis.yml

```diff
@@ -139,6 +139,22 @@ jobs:
             cd .ci && aws s3 cp . s3://$TRAVIS_S3_BUCKET/documentation-v2/$TRAVIS_BUILD_NUMBER --recursive --exclude "*.sh" --exclude "*.html" --exclude "*.yml" --exclude ".gitignore" && cd -
           fi;
 
+    - stage: Snippets tests
+      name: "Deploy Kuzzle with failling tests"
```

Comment:
Let's specify at least that we deploy the docs and not Kuzzle itself... "with failing tests" LOL

Reply:
```suggestion
      name: "Temporarily Deploy Kuzzle Docs with failling tests"
```